test/plots/barPlotTests.ts

```diff
@@ -788,23 +820,6 @@ describe("Plots", () => {
         plot.renderTo(svg);
       });
 
-      it("hides labels outside of the visible render area (horizontal)", () => {
```

Comment:
Curious though. Why is this necessary? Why make it `"hidden"` if it's already outside the viewport?